src/tree/DaprTreeItem.ts

```diff
@@ -22,7 +22,7 @@ export class DaprTreeItem extends AzExtTreeItem implements IAzureResourceTreeIte
         super(parent);
         this.label = localize('dapr', 'Dapr');
         this.data = data || {};
-        this.description = this.parent.data.template?.dapr ? 'Enabled' : 'Disabled';
+        // this.description = this.parent.data.template?.dapr ? 'Enabled' : 'Disabled';
```

Comment:
Why is the description excluded here?

Reply:
They changed where the dapr stuff lives now and I was too lazy to figure that out atm and fix it properly.

If they're going to change it on me, I'll just remove the tree entirely! (which I did.)

Reply:
Got it. I'm not sure how you're tracking issues like this one but it might be worth logging a GitHub issue on this repo  about dapr so comments like this don't get lost in the sauce